scripts.js

```diff
@@ -298,10 +319,28 @@ options.pan.addEventListener('click', function (event) {
     panToMarkers = event.target.checked;
 });
 
-options.precedence.addEventListener('change', (event) => {
+options.precedence.addEventListener('change', function (event) {
     setMarkerPrecedence(event.target.value == 'Bottom');
 });
 
+statistics.button.addEventListener('click', function() {
+    statistics.statistics.classList.toggle('open');
+});
+
+function loadStatistics(year) {
+    statistics.statistics.classList.remove('no-data');
+    statistics.class.textContent = year;
+    statistics.stats.students.textContent = students.get(year).length;
+
+    if (!stats.get(year)['Number of destinations']) {
+        statistics.statistics.classList.add('no-data');
+    }
+
+    statistics.stats.destinations.textContent = stats.get(year)['Number of destinations'];
+    statistics.stats.states.textContent = stats.get(year)['Number of states'];
+    statistics.stats.countries.textContent = stats.get(year)['Number of countries'];
+}
```

Comment:
We are adding a lot of new code here, and `scripts.js` has been getting bigger for a while now. Do you think this new statistics functionality could better be placed in a new file? (And more generally, should we consider splitting existing code from `scripts.js` into separate files?

Reply:
I like this idea, but I don't usually like including several JS files in the same HTML file and sharing functions between them (when we almost inevitably will need to). I think I'll do it for this stats stuff, but in the future maybe some kind of module system would work. I'll have to look into it.

Reply:
`util.js`?